tests/unit/cases_rst/only.rst

```diff
@@ -0,0 +1,6 @@
+.. only:: CS2
+
+   Tight Bounds
+   ~~~~~~~~~~~~
+
+   The definition for big-Oh..
```

Comment:
the result file is empty, is it ok?

Reply:
its OK. It serves to limit content by audience. The html version is also missing